### ISSUE TITLE
truffle test error when empty folder in test folder

### ISSUE THREAD

**Issue by eggplantzzz** [trufflesuite/truffle#2419]
When running `truffle test`, if an empty folder exists in the test directory it will fail with 
```
TypeError: Cannot read property 'files' of undefined
    at subloop (/Users/tyler/.nvm/versions/node/v8.16.0/lib/node_modules/truffle/build/webpack:/~/node-dir/lib/paths.js:92:1)
    at /Users/tyler/.nvm/versions/node/v8.16.0/lib/node_modules/truffle/build/webpack:/~/node-dir/lib/paths.js:107:1
    at onDirRead (/Users/tyler/.nvm/versions/node/v8.16.0/lib/node_modules/truffle/build/webpack:/~/node-dir/lib/path
.......
........
```

**Comment by eggplantzzz** [comment]
It turns out this issue is related to the `node-dir` library and unfortunately it has not been published for the last 2 years.

https://github.com/fshost/node-dir/issues/53

**Comment by gnidan** [comment]
good find!

**Comment by eggplantzzz** [comment]
This issue was resolved with the release of v5.0.40. https://github.com/trufflesuite/truffle/pull/2420
